superdesk/media/renditions.py

```diff
@@ -179,7 +181,7 @@ def _resize_image(content, size, format='png', keepProportions=True):
     assert isinstance(size, tuple)
     img = Image.open(content)
     width, height = img.size
-    new_width, new_height = [to_int(x) for x in size]
+    new_width, new_height = [int(x) for x in size]
```

Comment:
I don't think so, this is weight and size fro PIL, so I expect it to be int. I don't even think that int() casting is needed, but I kept because it was there before.